Childrens-Social-Care-CPD/wwwroot/stylesheets/custom.css

```diff
@@ -0,0 +1,296 @@
+﻿/* STANDARD COLOURS */
+body {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif !important;
+}
+
+.logo {
+    display: block;
+    max-height: 32px;
+    max-width: 100%;
+}
+/* SECTION CARDS ---------------- */
+/* shaded section card - used in current version */
+.section-card {
+    background-color: #f3f2f1;
+    padding: 15px;
+}
+
+.section-card__wborder {
+    background-color: #f3f2f1;
+    padding: 15px;
+    box-shadow: 0px 2px #1d70b8;
+}
+
+.section-card__wfullborder {
+    padding: 15px;
+    /*box-shadow: inset 0 0 0 2px $colour-drop-shadow;
+	*/
+    border: 2px solid #1d70b8;
+}
+
+.section-card__wbutton {
+    background-color: #f3f2f1;
+    padding: 30px;
+}
+
+.section-card__wimage img {
+    max-width: 100%;
+    margin-bottom: 20px;
+}
+
+.section-card__noborder {
+    background-color: #f3f2f1;
+    padding: 15px;
+    margin-bottom: 20px;
+}
+
+.section-card__clearwborder {
+    padding: 15px;
+    border: 1px solid #b1b4b6;
+}
+
+.section-card__clear {
+    padding: 15px;
+}
+/* CONTENT PAGES ---------------- */
+.aside-list__block {
+    margin-bottom: 50px;
+}
+
+.aside-list__indent {
+    padding-left: 10px;
+}
+
+.aside-list__heading {
+    font-size: 16px;
+    font-weight: bold;
+}
+
+.aside-list__related {
+    border-top: 3px solid #1d70b8;
+    padding-top: 20px;
+}
+
+    .aside-list__related:last-child {
+        border-bottom: 3px solid #1d70b8;
+        padding-top: 20px;
+    }
+
+.app-related-items {
+    border-top: 2px solid #347ca9;
+    padding-top: govuk-spacing(2);
```

Comment:
Does this property work without sass? I think it might be worth adding sass as most projects seem to use it and it's recommended. I would go through and change all of these properties to `govuk-spacing` and change all the colours to using `govuk-colour`. Also, using sass, you could probably make this file a lot shorter